ui/src/layouts/UserLayout.vue

```diff
@@ -37,8 +37,9 @@
       </div>
       <route-view></route-view>
     </div>
-    <div class="user-layout-footer" v-if="$config.loginFooter">
-      <label v-html="$config.loginFooter"></label>
+    <div class="user-layout-footer" v-if="$config.loginFooter || $config.resetPasswordFooter">
+      <label v-if="$route.name === 'resetPassword' && $config.resetPasswordFooter" v-html="$config.$config.resetPasswordFooter"></label>
+      <label v-else v-html="$config.$config.resetPasswordFooter"></label>
```

Comment:
should this be
```suggestion
      <label v-else v-html="$config.resetPasswordFooter"></label>
```
???